docs/source/notebooks/Image_Tut.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classification with Cleanlab, PyTorch & Skorch"
```

Comment:
[Nit] Better title might be: 
`Image Classification with PyTorch and Cleanlab`

I don't think Skorch is worth mentioning, and would try to stick with convention of `and Cleanlab` being at the end of the tutorial titles (since it is the least interesting part of the title given many of the tutorials will have this phrase in their title).

Reply:
done!